chrome.cast.js

```diff
@@ -491,6 +491,8 @@ chrome.cast.initialize = function (apiConfig, successCallback, errorCallback) {
 			_routeRefreshInterval = setInterval(function() {
 				execute('emitAllRoutes');
 			}, 15000);
+			
+			setTimeout(function() { execute('emitAllRoutes'); }, 2000);
```

Comment:
why is this needed? this delayed call...

Reply:
yes, exactly. I tried calling "execute('emitAllRoutes');" immediately on start, but it would not find any devices then, I guess it takes some time to fully initialize.

Reply:
Maybe try calling it on the `deviceready` event? https://cordova.apache.org/docs/en/8.x/cordova/events/events.html#deviceready

Reply:
Just tried it, seems to be too soon I guess?
I'm not a js/cordova developer, but I investigated a bit:
the method "chrome.cast.initialize" is being called by "CastPlayer.prototype.initializeCastPlayer" (JF web, src\jellyfin-web\src\components\chromecast\chromecastplayer.js). As far as I can see this in turn is called async (?) by ChromecastPlayer:
`castSenderApiLoader.load().then(initializeChromecast.bind(this));`
and may be called by itself after a delay of a second if chrome.cast is null or chrome.cast.isAvailable  is false.

Reply:
Having this triggered by an event might be more reliable but without being familiar with the codebase this seems like a reasonable approach.